online-prediction/predict/main.py

```diff
@@ -0,0 +1,69 @@
+"""
+Copyright (c) 2021. Cognitive Scale Inc. All rights reserved.
+
+Licensed under CognitiveScale Example Code [License](https://cognitivescale.github.io/cortex-fabric-examples/LICENSE.md)
+"""
+import logging
+import uvicorn
+import numpy as np
+import pandas as pd
+
+# cortex
+from cortex import Cortex
+
+# fastapi
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.post('/invoke')
+async def run(request: dict):
+    logging.info("Online Prediction: Invoke Request:{}".format(request))
+    columns = request['payload']["columns"]
+    instances = request['payload']["instances"]
+    df = pd.DataFrame(columns=columns, data=instances)
+    # Initialize Cortex Client
+    client = Cortex.client(api_endpoint=request["apiEndpoint"], token=request["token"], project=request["projectId"])
+
+    # Load Model from the experiment run
+    logging.info("Loading model artifacts from experiment run...")
+    model = await load_model(client, request["properties"]["experiment-name"], request["properties"]["run-id"])
+    logging.info("Model Loaded!")
+    try:
+        # If the model artifact is of type `dict`
+        if isinstance(model, dict):
+            cat_cols = model["cat_columns"] if "cat_columns" in model else []
+            num_cols = [x for x in df.columns if x not in cat_cols]
```

Comment:
better to use variables with proper names

Reply:
I used the variables to match with german credit train & predict example but yeah I can use the full names here.